### PR TITLE
update run workloads steps

### PR DESCRIPTION
**Pull request by ebattat**
Fix:

- Update the instruction to run workload through JupyterLab terminal.

### CODE REVIEW

File: benchmark_runner/jupyterlab/templates/run_workload/how_to_run_workload.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+update the repository with "git pull" if already cloned.
+```
+
+3. Change the directory to benchmark-runner:
+
+```
+cd benchmark-runner
+```
+
+4. **MUST**: Update the necessary environment variables in benchmark_runner/main/environment_variables.py, e.g., ('WORKLOAD', 'vdbench_vm').
```
Comment (RobertKrawitz) [inline benchmark_runner/jupyterlab/templates/run_workload/how_to_run_workload.md:23]: Modifying a file just to run a command isn't a great idea.  Either set the environment variables in the environment or use command line options rather than modifying this file.

File: benchmark_runner/jupyterlab/templates/run_workload/how_to_run_workload.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+5. For custom workload parameters, configure the templates in benchmark_runner/common/template_operations/templates.
+
+6. Run the following command in the terminal:
+
+```
+PYTHONPATH=. python3 benchmark_runner/main/main.py "$@"
```
Comment (RobertKrawitz) [inline benchmark_runner/jupyterlab/templates/run_workload/how_to_run_workload.md:30]: The `"$@"` is used for processing command line arguments.

Why did you delete run_workload.sh?  It's simpler to run `run_workload.sh` than to issue a python command line with PYTHONPATH set.

**Review by RobertKrawitz — CHANGES_REQUESTED** [review]

File: benchmark_runner/jupyterlab/templates/run_workload/how_to_run_workload.md (hunk 2 above)
Comment (ebattat) [reply benchmark_runner/jupyterlab/templates/run_workload/how_to_run_workload.md:26]: You right, see the updated code

**Review by RobertKrawitz — CHANGES_REQUESTED** [review]

File: benchmark_runner/jupyterlab/templates/run_workload/run_workload.sh [hunk 3]
```diff
@@ -1,5 +1,3 @@
-# Follow the stops:
-# git clone https://github.com/redhat-performance/benchmark-runner
-# update environment variables i.e. ('WORKLOAD','vdbench_vm') in benchmark_runner/main/environment_variables.py
-# run from benchmark-runner root path
-PYTHONPATH=. python3 benchmark_runner/main/main.py "$@"
+!/bin/sh
+
+PYTHONPATH=. python3 benchmark_runner/main/main.py "$@"
```
[the remark this answers is not in the document]
Comment (ebattat) [reply benchmark_runner/jupyterlab/templates/run_workload/run_workload.sh:3]: fixed

**Review by RobertKrawitz — APPROVED** [review]
